app/javascript/src/assets/scss/widgets/_tabs.scss

```diff
@@ -1,50 +1,52 @@
 .tabs {
-  border-width: 0;
   @include padding($zero $space-normal);
   @include border-normal-bottom;
+  border-width: 0;
```

Comment:
^ might need to remove the above line, this removes the border as specified in other PR.

Reply:
Can you please check it new PR. If it's necessary, I wil fix it.

Reply:
#125 this is new PR